pkg/role/role.go

```diff
@@ -0,0 +1,32 @@
+/*
+Package role manages roles in Kvdb and provides validation
+Copyright 2018 Portworx
```

Comment:
Is this ok since we technically don't own openstorage.

Reply:
Yeah, that is fine. At Red Hat, we used `Copyright <year> Red Hat` in the same manner since we worked for Red Hat. The company has the copyright